xtra-libp2p/src/dialer.rs

```diff
@@ -36,7 +36,8 @@ impl Actor {
         }
     }
 
-    /// Returns error if we cannot access Endpoint or the peer is not connected Endpoint.
+    /// Returns error if we cannot access the Endpoint or if the peer is not
+    /// connected to the Endpoint.
```

Comment:
Does `dprint check` pass for all commits before this one? You are not changing the comment here.

Reply:
I tried checking running `dprint` after each commit and couldn't find anything.
Also, I don't know how to refer back to particular commit from github UI, so I can't tell for sure.